### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by MrAdityaAlok**
Update outdated packages

TODO:
- [x] apache2
- [x] gradle
- [x] age
- [x] asciidoctor
- [x] at
- [x] bitcoin
- [x] boost

### CODE REVIEW

File: packages/apache2/build.sh [hunk 1]
```diff
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://httpd.apache.org
 TERMUX_PKG_DESCRIPTION="Apache Web Server"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=1:2.4.48
-TERMUX_PKG_SRCURL=https://www.apache.org/dist/httpd/httpd-${TERMUX_PKG_VERSION:2}.tar.bz2
-TERMUX_PKG_SHA256=1bc826e7b2e88108c7e4bf43c026636f77a41d849cfb667aa7b5c0b86dbf966c
+TERMUX_PKG_VERSION=2.4.49
```
Comment (thunder-coding) [inline packages/apache2/build.sh:5]: Why did you remove the package epoch here? The package manager will ignore this version because it has a lower epoch than the old version

**Review by thunder-coding — CHANGES_REQUESTED** [review]

File: packages/apache2/build.sh (hunk 1 above)
Comment (MrAdityaAlok) [reply packages/apache2/build.sh]: > Why did you remove the package epoch here?
By mistake 😅

**Comment by MrAdityaAlok** [comment]
Closing in favour of #7572
